src/payment/data/sagas.test.js

```diff
@@ -311,8 +421,8 @@ describe('saga tests', () => {
       );
     });
 
-    it('should show a fallback error message', async () => {
-      response = { basket: Factory.build('basket', {}, { numProducts: 1 }) };
+    it('should update basket data and show an error message', async () => {
```

Comment:
Added this test to prove we call basketDataReceived with catchBasketError if we have a basket. and an error.